### ISSUE TITLE
Beta1 labeling

### ISSUE THREAD

**Issue by ppf2** [elastic/sense#79]
As of today, the latest Sense release is 2.0-beta1.  The "beta" tagging is causing some confusion in the field for the end user expects the version 2.0 to match up with ES 2.0.  Pretty sure this is expected today, so just creating this ticket to discuss if we can sync up Sense and ES versioning for future releases.

**Comment by bleskes** [comment]
Yeah, 2.0 GA is coming up and we will follow the ES releases in the future.
